extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -1,22 +1,38 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
-use \Wikia\Tasks\Tasks\ImageReviewTask;
+use Wikia\Logger\WikiaLogger;
+use Wikia\Rabbit\ConnectionBase;
+use Wikia\Tasks\Tasks\ImageReviewTask;
 
 class ImageReviewEventsHooks {
 	public static function onUploadComplete( UploadBase $form ) {
-		static::createAddTask( $form->getTitle() );
+		global $wgCityId, $wgImageReviewTestCommunities;
+
+		// TODO: community switch
+		//if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
+			self::sendToImageReviewService( $form->getTitle() );
+		//} else {
+		//	static::createAddTask( $form->getTitle() );
+		//}
 
 		return true;
 	}
 
 	public static function onFileRevertComplete( Page $page ) {
-		static::createAddTask( $page->getTitle() );
+		global $wgCityId, $wgImageReviewTestCommunities;
+
+		// TODO: community switch
+		//if ( in_array( $wgCityId, $wgImageReviewTestCommunities ) ) {
+		self::sendToImageReviewService( $page->getTitle() );
+		//} else {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)